### PR TITLE
Convert latest samples to .NET Core.

### PR DESCRIPTION
**Pull request by cesarblum**
- [x] Ensure samples in the `latest` folder work with `dotnet restore` and `dotnet run`.
- [x] Fix Dockerfiles.
- [ ] Fix xproj files.

cc @Eilon @muratg @victorhurdugaci @davidfowl

### CODE REVIEW

**Comment by muratg** [comment]
@glennc

File: samples/latest/HelloMvc/NuGet.Config [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetvnext/api/v2" />
+    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v2" />
```
Comment (cesarbs) [inline samples/latest/HelloMvc/NuGet.Config]: What do you guys think of switching to `aspnetcidev` for the samples in `latest`?

Comment (Eilon) [inline samples/latest/HelloMvc/NuGet.Config]: Ideally this should stay `aspnetvnext`... the samples here should reflect the last overall good build. Though right now we're in the unfortunate state where we haven't had good builds...

**Comment by moozzyk** [comment]
I wonder whether we shouldn't merge this with https://github.com/aspnet/cli-samples/

**Comment by cesarbs** [comment]
Samples working on Docker now but still need to use `cidev` feed.

**Pull request closed without merging**